src/framework/components/camera/component.js

```diff
@@ -91,6 +91,7 @@ Object.assign(pc, function () {
         this.on("set_cullFaces", this.onSetCullFaces, this);
         this.on("set_flipFaces", this.onSetFlipFaces, this);
         this.on("set_layers", this.onSetLayers, this);
+        this.system.app.on("prerender", this.onPrerender, this);

```

Comment:
This adds a listener for every camera in the hierarchy. I think we can do the same job but in the camera/system.js so we only have one event listener which loops through all cameras.